web/src/app/modules/views/main/editors/modules/graphical-editor/components/graphical-editor.component.html

```diff
@@ -27,6 +27,7 @@ <h5 class="card-header">{{name}}
                     [ngStyle]="{'background': 'url(/assets/img/grid.png)', 'background-size': editorGridButton.getBackgroundSize()}">
                 </div>
             </div>
+            <div id="renderingContainer"></div>

```

Comment:
At the moment it is above the original, thus the dots are visible for the time of saving. Put it under the original

![image](https://user-images.githubusercontent.com/49229000/105713368-5a611680-5f1b-11eb-84d6-a9a632584cb7.png)